pallets/rolldown/rpc/src/lib.rs

```
@@ -71,4 +80,21 @@ where
 			)))
 		})
 	}
+
+	fn verify_pending_requests(
+		&self,
+		hash: H256,
+		request_id: u128,
+		at: Option<<Block as BlockT>::Hash>,
+	) -> RpcResult<bool> {
+		let api = self.client.runtime_api();
+		let at = at.unwrap_or(self.client.info().best_hash);
+		api.verify_pending_requests(at, hash, request_id).map_err(|e| {
```

once verify_pending_requests returns Option 

api.verify_pending_requests witll return Result<Opion<bool>>

first result is set to error when there is some problem with runtime api execution, in that case we should raise error
When  option is set to None we also should raise an error because that means requested request_id does not exists